cpp/epidemiology_io/json_serializer.h

```diff
@@ -601,15 +601,15 @@ IOResult<T> read_json(const std::string& path, Tag<T> tag, int flags = IOF_None)
 //Implementations for JsonContext/Object member functions below//
 /////////////////////////////////////////////////////////////////
 
-template <class T, std::enable_if_t<JsonType<T>::value, void*> = nullptr>
```

Comment:
I don't know if this is correct SFINAE anymore.

@dabele Could you check that everyting still is as intended?

Reply:
It is. MSVC gives me a "warning treated as error", because the default template argument was doubly defined: 

https://github.com/DLR-SC/memilio/blob/81b9a1494820e1c6d9c2f34aad4f957a4aa66acd/cpp/epidemiology_io/json_serializer.h#L280

It suffices to define it in the definition.